ports/zephyr/README.md

```diff
@@ -63,6 +63,11 @@ To build for QEMU instead:
 
     $ west build -b qemu_x86 ~/micropython/ports/zephyr
 
+To build any board with the `_thread` module enabled, 
+add `-DOVERLAY_CONFIG=thread.conf`, for instance:
+
+$ west build -b frdm_k64f ~/micropython/ports/zephyr -DOVERLAY_CONFIG=thread.conf
```

Comment:
Please indent this line with 4 spaces.

Also, might be worth replacing `~/micropython/ports/zephyr` with `--` so it works regardless of where the MP repo is (but then this example would be inconsistent with other examples in this file... probably worth changing them all).